src/store/modules/Authentication/AuthenticanStore.js

```diff
@@ -42,6 +42,7 @@ const AuthenticationStore = {
       api
         .post('/logout', { data: [] })
         .then(() => commit('logout'))
+        .then(() => (window.location = '/logout'))
```

Comment:
Can you explain why this works?

Reply:
That is right, using the vue route did not reset the store, which I feel should be reset on logout. Posting to the logout REST API, I believe, expires the session on the back end. It does not redirect us to the logout screen or clear the application store. 

I believe that creates a potential security risk. It is possible to use the back button and see information of the last page if the store is not reset. The easiest way to reset the store is a full page post back.

Reply:
`window.location = '/login'` ?

Reply:
Yes...You're correct. It should be `login`. It works because of the navigation guard, but it should redirect to login. Thank you!

Reply:
All set, just  pushed that update. Are you good with the full post back to clear the store?

Reply:
makes sense to me...otherwise we'd have to manually clear the store state right? Does Vuex have any documentation/recommendations around handling this?

Reply:
I researched and found that the full post back is the easiest way and given that we are logging the user out, it makes sense to me. Otherwise, I found the following conversations about resetting the state of the entire store:

1. https://github.com/vuejs/vuex/issues/1118
1. https://forum.vuejs.org/t/how-to-completely-reset-the-vuex-store/13611/5
1. https://stackoverflow.com/questions/42295340/how-to-clear-state-in-vuex-store

I am good if we want to take this approach as well. I don't know any other use case right now that would require us to reset all the modules.